content/logo/index.mdx

```diff
@@ -0,0 +1,138 @@
+---
```

Comment:
For this one, the route is only working on `/logos/index`, so you would have to change the `getStaticPaths` and `getStaticProps` methods in `pages/logo/[filename].tsx` to map this to `/`

Reply:
Done. Page now mapping to `/logo` instead of `/logo/index`